ui/scss/user-dashboard/_sites.scss

```diff
@@ -68,6 +68,10 @@
 
 }
 
+.green .modal-body .notificationBody .icon{
+  color: #28a745;
```

Comment:
Use `#34C759`. Please put it on a variable for reusability.

Reply:
Done